pmd-xml/src/main/resources/category/pom/codestyle.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0"?>
+
+<ruleset name="Codestyle"
+         xmlns="http://pmd.sourceforge.net/ruleset/2.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://pmd.sourceforge.net/ruleset/2.0.0 http://pmd.sourceforge.net/ruleset_2_0_0.xsd">
+
+    <description>
+Rules which enforce a specific coding style.
+    </description>
+
+    <rule name="ProjectVersionAsDependencyVersion"
```

Comment:
is this really a style issue or an errorprone / bestpractice?

> Using that expression in dependency declarations seems like a shortcut, but it can go wrong. By far the most common problem is the use of ${project.version} in a BOM or parent POM.

Reply:
The rule is very broad and causes currently lots of false positives: It basically prevents using ${project.version} at all. Looking at the example, I think, what the intention was, is this: Never use project.version for a **unrelated** dependency - that is, e.g. a dependency with a complete different groupId. I'll move it to errorprone and create a new issue to improve this rule for 6.1.0.

Reply:
Agreed. Sounds reasonable.

Reply:
See #715